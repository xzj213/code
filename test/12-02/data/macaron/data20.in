14
0 585 585 51 0 51 51 0 51 585 585 585 51 51
140 585 0 51 51 585 51 0 51 140 140 51 51 585
140 51 585 0 0 585 51 0 585 585 51 585 585 51
51 51 585 0 585 585 140 585 754 0 585 51 585 140
140 0 0 140 140 140 0 140 140 0 140 585 51 585
585 0 585 585 51 51 585 140 140 0 140 585 585 140
0 0 585 585 51 140 140 51 0 140 585 140 0 51
272 140 585 140 585 51 585 585 0 51 51 140 140 0
1023 585 51 140 140 0 51 140 284 585 51 140 51 140
140 140 140 140 585 585 140 0 585 585 140 0 51 51
585 0 51 140 585 51 140 0 0 0 140 51 0 0
140 0 0 585 140 0 51 602 585 585 51 51 585 0
140 140 140 585 51 0 51 585 51 51 585 51 140 0
0 0 51 0 51 140 585 585 51 140 140 140 140 0
