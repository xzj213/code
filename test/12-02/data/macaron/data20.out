0 4 12 19 23 31 32 36 44 51 55 63 128 136 140 147 155 159 160 168 172 179 187 191 259 263 265 269 271 272 276 282 284 286 291 295 297 301 303 304 308 314 316 318 385 387 389 395 399 400 402 406 408 412 417 419 421 427 431 432 434 438 440 444 581 585 589 598 602 606 613 617 621 630 634 638 705 709 713 722 726 730 737 741 745 754 758 762 832 836 838 842 846 851 853 855 857 861 864 868 870 874 878 883 885 887 889 893 962 966 968 970 972 977 981 985 987 991 994 998 1000 1002 1004 1009 1013 1017 1019 1023 
